### ISSUE TITLE
Update blog post on kubernetes 

### ISSUE THREAD

**Issue by nabadger** [luraproject/lura#247]
I was following the post here https://www.krakend.io/blog/krakend-on-kubernetes/ and have a couple of recommendations.

1 - It references an old version of krakend `krakend:0.4.2`. I think there must be numerous bug fixes since latest, because I was running into an issue with encoding and wondering why various encoding settings were not resolving an issue I was having. Finally looked at releases, bumped to the latest version and voila, it works :)

2 - I generally find that the config should be done as a k8s `ConfigMap` resource.

That gives a couple of benefits

- It probably simplifies the document, since you don't need to describe how to build your own dockerfile with the bundled config.

- It's easier it work with in general and seems to be the k8s "native" way of doing things (it saves the end user-time in having to re-publish a dockerfile too).

Thanks for this software :)

**Comment by alombarte** [comment]
Thanks @nabadger 

When we wrote the post we used the KrakenD version that we had at that time, but since the article continues being valid today, I already removed the `:0.4.2` tag from the `Dockerfile` in the article. Good catch!

As per the `ConfigMap`, if you have done it already, feel free to provide the resulting code, and I will attach it to the article (with your credit). The configuration we used was using the same pattern provided in the k8s docs when we did it, the idea was that users were familiar with it. But I am sure that your comment is pointing the right way to do it and I have no inconvenience in updating it as well, but believe me that I do not have much time for this so any help is greatly appreciated.
